.github/workflows/ci.yml

```
@@ -43,7 +43,7 @@ jobs:
       - uses: julia-actions/julia-buildpkg@v1
       - uses: julia-actions/julia-runtest@v1
       - uses: julia-actions/julia-processcoverage@v1
-      - uses: codecov/codecov-action@v3
+      - uses: codecov/codecov-action@v4
         with:
           file: lcov.info
```

```suggestion
        with:
          token: ${{ secrets.CODECOV_TOKEN }} # required
          fail_ci_if_error: true
          file: lcov.info
```
xref: [JuliaMath/AbstractFFTs.jl/pull/129](https://github.com/JuliaMath/AbstractFFTs.jl/pull/129)